modules/databases/postgresql_flexible_server/server.tf

```diff
@@ -14,7 +14,7 @@ resource "azurerm_postgresql_flexible_server" "postgresql" {
   location            = var.location
   version             = try(var.settings.version, null)
   sku_name            = try(var.settings.sku_name, null)
-  zone                = try(var.settings.zone, null)
+  zone                = try(var.settings.zone, 1)
```

Comment:
Waarom is dit nu 1? Is dit een ok default value? Of is null goed zodat hij weggelaten word indien niet gezet?